Dockerfile

```diff
@@ -1,43 +1,38 @@
-FROM fedora:22
-MAINTAINER Vaclav Pavlin <vpavlin@redhat.com>
+FROM fedora
+MAINTAINER Developer-portal <developer-portal@lists.fedoraproject.org>
 
 # Gems require ruby-devel and group C Development Tools and Libraries
 # We need git to get a website and a content of Developer Portal
 # Nodejs is needed by Jekyll and iproute provides ip command which let's us to set host for Jekyll properly
```

Comment:
Is nodejs really needed by jekyll?

Also, see the iproute comment.

Reply:
The nodejs is installed by setup.sh
I just tried to run it without nodejs and it's working

Reply:
is there anything wrong with the iproute comment?

Reply:
I'd prefer to remove nodejs from `setup.sh` then, to avoid unnecessary dependencies (pleasase make sure the search works, as it can be used to generate json etc.).

iproute - nothing, it's just that it was mentioned as a dependency, and also installed using docker RUN:
  `dnf -y install ruby-devel git nodejs iproute zlib-devel libxml2 ...`

Reply:
tested it now, searching works just fine without nodejs